style.css

```diff
@@ -766,4 +951,40 @@ form {
     gap: 3rem;
     font-size: 10px;
   }
+
+  .buttout {
+    padding: 8px 20px;
+    background: transparent;
+    color: #6070ff;
+    border: 1.5px solid #6070ff;
+    border-radius: 6px;
+    margin: 10px 12px;
+    text-align-last: justify;
+  }
```

Comment:
- [ ] Please modify the buttons with the appropriate padding and size so that it looks like the figma design
![image](https://user-images.githubusercontent.com/46839548/224188712-f8d357b9-ba65-490d-abb4-9b5f5b4b06f3.png)